code/modules/surgery/robotics.dm

```diff
@@ -8,17 +8,35 @@
 
 /datum/surgery_step/robotics/can_use(mob/living/user, mob/living/carbon/human/target, target_zone, obj/item/tool)
 	if (isslime(target))
+		#ifdef SURGERYDEBUG
+		log_debug("surgery_step/robotics/can_use Surgery step returned 0; target is slime")
```

Comment:
Do we need to leave these debug logs in? They're kinda... a lot.

Reply:
I recommend leaving it in because stuff is legit about the only way you're going to debug surgery code, but I can remove them if asked.

Reply:
Updated to remove all debugging.

Reply:
Thanks. I appreciate that debug text messages are frequently the way to go in ss13, I think leaving them lying around like this hurts code readability overall.